thunder/executors/torch_autograd.py

```diff
@@ -240,6 +240,23 @@ def split_forward_backward(computation_trc: TraceCtx, compile_data, compile_stat
         skip_output=False,
         skip_subsymbols=False,
     )
+
+    # remove duplicates
+    # The NVFuser (and possibly others) fusion pass applied on the forward during has a
```

Comment:
This is super interesting!

fyi @jjsjann123 @kshitij12345 @IvanYashchuk

Reply:
Out of curiosity, do duplicates cause any functional issue?